pykern/pkresource.py

```diff
@@ -8,34 +8,82 @@
 
 # Root module: Import only builtin packages so avoid dependency issues
 import errno
+import glob as builtin_glob
```

Comment:
@robnagler thoughts on using as?

Reply:
This is a smell. Rename your method (see https://github.com/radiasoft/sirepo/pull/3747#discussion_r685268361) to sorted_glob or something that doesn't collide.

Reply:
ccca032